knpu/ep1-intro/listing-services.md

```diff
@@ -1,16 +1,18 @@
 # Listing and Using Services
 
 Rendering a template is pretty common, so there's a shortcut when you're in a controller.
-Replace all of this code with a simple `return $this->render`.
+Replace all of this code with a simple `return $this->render`:
+
+[[[ code('6ad09ec8e1') ]]]
 
 That's it. Make sure this works by refreshing.
 
 So what does this magic-looking `render()` function actually do? Let's find out!
-Hold command or control (depending on your OS) and click `render` to be taken straight
+Hold command or control (depending on your OS) and click `render()` to be taken straight
 into the base `Controller` class where this function lives: deep in the heart of Symfony.
 
```

Comment:
Ah, you're right! it's a little weird :). I don't think it's a perfect solution yet, but I added *just* this `Controller.php` file into the master branch, exactly where it will appear later if you move to a step with a `composer.json` file and run `composer install`. Then I added the code block for it. It would be better if the `browser` app were also able to deliver files from the `vendor/` directory without us needing to commit them.